qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/QuteDiagnostics.java

```diff
@@ -532,10 +524,15 @@ private ResolvedJavaTypeInfo validateExpressionParts(Parts parts, Section ownerS
 					// {items.size()}
 					// Property, method to validate must be done for iterable type (ex :
 					// java.util.List
-					String iterableType = baseType.getIterableType();
-					baseType = resolveJavaType(iterableType, projectUri, resolvingJavaTypeContext);
-					if (baseType == null || isResolvingJavaType(baseType)) {
-						// The java type doesn't exists or it is resolving, stop the validation
+					// String iterableType = baseType.getIterableType();

```

Comment:
If this code in the comments is unused, remove it

Reply:
fixed